### PR TITLE
#2245 Stream APIs take multiples from/to topics but the streamapp tak… (backport to 0.7)

### PR DESCRIPTION
**Pull request by saivirtue**
…e them as single from/to topic (backport to 0.7)

### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2245)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: docs/src/rest/rest-stream.rst [hunk 1]
```diff
@@ -109,7 +109,17 @@ Example Request
 
   #. jmxPort (**int**) — expose port for jmx ; default is random port
   #. from (**array(string)**) — source topic ; default is empty array
+      .. note::
+       we only support one topic for current version. We will throw exception in start api if you assign
+       more than 1 topic.
+
+      [TODO] We will support multiple topics on issue :ohara-issue:`688`
```
Comment (chia7712) [inline docs/src/rest/rest-stream.rst]: Do you plan to backport multiple from/to topics to 0.7? If not, please say sorry to 0.7 user ;(

Comment (saivirtue) [reply docs/src/rest/rest-stream.rst]: Oops...

**Comment by chia7712** [comment]
@saivirtue Let me rebase this PR for you and QA

**Comment by chia7712** [comment]
@wu87988622 Please take a look at this PR. thanks

**Comment by wu87988622** [comment]
> @wu87988622 Please take a look at this PR. thanks

ok~

**Pull request merged**